SECURITY_CONTACTS

```diff
@@ -4,8 +4,7 @@
 # to for triaging and handling of incoming issues.
 #
 # The below names agree to abide by the
-# [Embargo Policy](https://github.com/kubernetes/sig-release/blob/master/security-release-process-document
-ation/security-release-process.md#embargo-policy)
+# [Embargo Policy](https://git.k8s.io/security/security-release-process.md)
```

Comment:
There is nothing about the embargo policy under this link, is this the correct one? If so, maybe rename it to "Security Release Process"?

Reply:
@markgeraty could you please follow up?

Reply:
I will merge this PR and fix link name in separate one